README.md

```diff
@@ -208,6 +216,14 @@ Generated by [@ucanto/principal `EdSigner`](https://github.com/web3-storage/ucan
 
 _Example:_ `MgCZG7EvaA...1pX9as=`
 
+#### `CONTENT_CLAIMS_PRIVATE_KEY`
+
+The [`multibase`](https://github.com/multiformats/multibase) encoded ED25519 keypair used as the signing key for the [content-claims api](https://github.com/web3-storage/content-claims).
```

Comment:
Should mention that it is `base64pad` encoded since there are many multibases 😅

Also, it doesn't _have_ to be the private key of the content claims service - it can be any private key with a valid delegation (see my other comment).

```suggestion
The `base64pad` [`multibase`](https://github.com/multiformats/multibase) encoded ED25519 keypair used as the signing key for [content-claims](https://github.com/web3-storage/content-claims).
```